hdf5.spec

```
@@ -241,19 +239,12 @@ HDF5 tests with openmpi3
 %endif
 
 %prep
-%setup -q -a 2 -n %{name}-%{hdf5_commit}
+%setup -q -a 2 -n %{name}-%{name}-%{version}-%{hdf5_release}
```

```suggestion
%setup -q -a 2 -n %{name}-%{name}-%{hdf5_major}_%{hdf5_minor}_%{hdf5_bugfix}%{?hdf5_prerelease:-%{hdf5_prerelease}
```

It's probably worth drying out that `%{hdf5_major}_%{hdf5_minor}_%{hdf5_bugfix}%{?hdf5_prerelease:-%{hdf5_prerelease}` above into a single macro which can then be reused here, in the `Version:`, etc.